File: src/plugin/database/models/PlayerPrefs.java
package plugin.database.models;

public class PlayerPrefs {
    public boolean showWelcomeMenu = true;
    public String customName = "";

    public void setShowWelcomeMenu(boolean showWelcomeMenu) {
        this.showWelcomeMenu = showWelcomeMenu;
    }

    public void setCustomName(String s) {
        this.customName = s;
    }
}

File: src/main/kotlin/plugin/database/models/PlayerPrefs.kt
package plugin.database.models

import com.fasterxml.jackson.annotation.JsonIgnoreProperties

@JsonIgnoreProperties(ignoreUnknown = true)
class PlayerPrefs(
    var showWelcomeMenu: Boolean = true,
    var owoAccent: Boolean = false,
    var ohioAccent: Boolean = false
)